app/registries/foreman/plugin/rbac_support.rb

```diff
@@ -34,7 +37,7 @@ def add_permissions_to_default_roles(roles)
 
       def check_role_name_before_extending(role_name)
         msg = "Invalid role name, only 'Viewer' or 'Manager' is allowed to be extended from a plugin"
-        raise Foreman::Exception.new msg if role_name != "Viewer" && role_name != "Manager"
+        raise Foreman::Exception.new msg if role_name != "Viewer" && role_name != "Manager" && role_name != "Organization admin"
```

Comment:
I kept the constant commit separate, if we'll cherry-pick this into stable branch we might want to keep it minimal. I can squash if you're OK with taking both to 1.15.2.